packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -183,6 +185,23 @@ class DropdownMenu<T> extends StatefulWidget {
   /// Defaults to null;
   final String? hintText;
 
+  /// Text that provides context about the [DropdownMenu]'s value, such
+  /// as how the value will be used.
+  ///
+  /// If non-null, the text is displayed below the input field, in
+  /// the same location as [errorText]. If a non-null [errorText] value is
+  /// specified then the helper text is not shown.
+  ///
+  /// Defaults to null;
+  final String? helperText;
+
+  /// Text that appears below the input field and the border to show the error message.
+  ///
+  /// If non-null, the border's color animates to red and the [helperText] is not shown.
+  ///
+  /// Defaults to null;
+  final String? errorText;
```

Comment:
Nit: For both of these new parameters, would it be useful to link to the corresponding InputDecorator property or is that too much detail?

```
/// See also:
///
///  * [InputDecorator.errorText]
```

Reply:
Agree with @justinmc :) Might be more clear to users if we can put this info there. I think we can do something like this:
```dart
///
/// See also:
/// 
/// * [InputDecoration.errorText], which is the text that appears below the [InputDecorator.child] and the border.
```

For helper text, we can put something like:
```dart
///
/// See also:
/// 
/// * [InputDecoration.helperText], which is the text that provides context about the [InputDecorator.child]'s value.
```